hack/lib/swagger.sh

```diff
@@ -50,6 +49,7 @@ kube::swagger::gen_types_swagger_doc() {
 
 // AUTO-GENERATED FUNCTIONS START HERE. DO NOT EDIT.
 EOF
+  } > "${TMPFILE}"
```

Comment:
this is mostly equivalent but more efficient (only opens & redirects once) and shellcheck prefers grouping commands when you appear to be redirecting many commands to the same place

Reply:
we could turn off the lint instead, but this is actually a minor improvement